docs/rfc/rfc-002-optimistic-execution.md

```diff
@@ -0,0 +1,120 @@
+# RFC 002: Optimistic Execution
```

Comment:
This number should change.

Reply:
Are we tracking what's the latest somewhere? Or should I check among open PRs?

Reply:
It should be tracked here: https://docs.cosmos.network/main/rfc. Probably the table content of that page needs to be as well updated.

Reply:
lets modify this to 5